### PR TITLE
Release 4.15.2

### PR DESCRIPTION
**Pull request by alexandrebodin**
### What does it do?

Describe the technical changes you did.

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

**Review by markkaylor — APPROVED** [review]
Thanks for handling this and publishing the one package I didn't have permissions on 😄

File: packages/core/admin/_internal/node/staticFiles.ts [hunk 1]
```diff
@@ -26,7 +27,7 @@ const getEntryModule = ({ plugins }: EntryModuleArgs): string => {
          * Any modifications made will be discarded.
          */
         ${pluginsImport}
-        import { renderAdmin } from "@strapi/admin/strapi-admin"
+        import { renderAdmin } from "@strapi/strapi/admin"
```
Comment (markkaylor) [inline packages/core/admin/_internal/node/staticFiles.ts:30]: @joshuaellis The CI is failing here for `yarn build:size` https://github.com/strapi/strapi/actions/runs/6802714709/job/18519164330#step:6:230, and I see it was also failing on your PR where this change was introduced: https://github.com/strapi/strapi/actions/runs/6744780346/job/18335312467

I am unable to reproduce the error locally. Any idea what might be the issue?

Comment (joshuaellis) [reply packages/core/admin/_internal/node/staticFiles.ts:30]: Differences in how things are built across the branches - imo we can ignore this & merge. 

If it pops up post merge then lmk and I'll take a look 😎

Comment (alexandrebodin) [reply packages/core/admin/_internal/node/staticFiles.ts:30]: Ok I'll force merge. Can you check on main once it's done please.

**Pull request merged**